keyword_checker.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+    /*
+    * Scans input string for convention violations and returns feedback based on
+    * detected violations
+    * Returns a string containing the formatted feedback message
+    */
+    getFeedback: function (string) {
+        message = "";
+        
+        if (containsSpaceBetweenHashtagAndDigit(string)) {
+            console.log("Detected space between # and digit");
+            message += getFormattedMessage(getMessageSpaceBetweenHashtagAndDigit());
+        } else if (!containsIssueReference(string)) {
+            console.log("Issue reference not found");
+            message += getFormattedMessage(getMessageIssueReferenceMissing());
+        }
+        
+        if (!containsGithubKeyword(string)) {
+            console.log("Missing GitHub keyword");
+            message += getFormattedMessage(getMessageGithubKeywordMissing());
+        }
+        
+        return message;
+    }
+};
+
+/*
+* Returns true if argument contains an issue reference, false otherwise
+*/
+function containsIssueReference(string) {
+    referenceTest = new RegExp(/[\s\S]*#\d[\s\S]*/);
```

Comment:
Also, try to extract out the actual regexp as a function. That way, you don't really repeat the code multiple times.